numpy/_core/tests/examples/cython/limited_api.pyx

```diff
@@ -0,0 +1,11 @@
+#cython: language_level=3
+
+"""
+Make sure cython can compile in limited API mode (see meson.build)
```

Comment:
Can you extend this a bit? I'm a little confused about why this was only a 2-line fix and when this applies - e.g., only if there's no `cimport numpy`?

Reply:
Currently `cimport numpy` generates some invalid code in the limited api on the Cython end.

I have a proposed patch but it isn't a particularly simple fixed so it's not yet merged. And will probably only make it into Cython 3.1 which you won't be testing with yet (because it isn't released....).

To be honest, at this stage it might be better to exclude Cython from this, and just compile a C header with these two includes and `Py_LIMITED_API` defined.